modules/plot_circular_path.py

```diff
@@ -0,0 +1,95 @@
+"""
+Module to plot n circular waypoints, given a center and radius.
+"""
+
+import csv
+import math
+
+from modules.common.mavlink.modules.drone_odometry import DronePosition
+from modules.common.mavlink.modules.drone_odometry_local import DronePositionLocal
+from modules.common.mavlink.modules.local_global_conversion import drone_position_global_from_local
+
+from .waypoint import Waypoint
+
+
+def move_coordinates_by_offset(
+    start_point: Waypoint, offset_x: float, offset_y: float, name: str
+) -> Waypoint:
```

Comment:
Let's make this return `"tuple[bool, Waypoint | None]"` for more safeguarding. In case any of the suboperations fail. (I only noticed this later, I have a more detailed comment below)